app/scripts/modules/amazon/src/instance/awsInstanceType.service.js

```diff
@@ -8,34 +8,34 @@ import { API } from '@spinnaker/core';
 module.exports = angular
   .module('spinnaker.amazon.instanceType.service', [])
   .factory('awsInstanceTypeService', function($http, $q) {
-    var m4 = {
-      type: 'm4',
+    var m5 = {
+      type: 'm5',
       description:
-        'This family includes the m4 instance types and provides a balance of compute, memory, and network resources, and it is a good choice for many applications.',
+        'm5 instances provide a balance of compute, memory, and network resources. They are a good choice for most applications.',
```

Comment:
changed from `many` to `most` applications as a result of some feedback/learning we've gotten internally.